tests/e2e/helpers/keys.ts

```diff
@@ -42,22 +43,104 @@ export const keyTypes = [
  * @param count The count of keys to add
  * @param keyNameStartWith The name of the key
  */
-export async function populateDBWithHashes(host: string, port: string, count: number, keyNameStartWith: string): Promise<void> {
+export async function populateDBWithHashes(host: string, port: string, keyArguments: AddKeyArguments): Promise<void> {
     const dbConf = { host, port: Number(port) };
     const client = createClient(dbConf);
 
-    await client.on('error', async function(error: string) {
+    await client.on('error', async function (error: string) {
         throw new Error(error);
     });
-    await client.on('connect', async function() {
-        for (let i = 0; i < count; i++) {
-            const keyName = `${keyNameStartWith}${common.generateWord(20)}`;
-            await client.hset([keyName, 'field1', 'Hello'], async(error: string) => {
-                if (error) {
-                    throw error;
-                }
-            });
+    await client.on('connect', async function () {
```

Comment:
I recommend to close the connection in the end of each function

Reply:
added client.quit(),, thanks!